doc/code_style/mom.cpp

```diff
@@ -59,34 +59,32 @@
  * Will be parsed by Doxygen.
  */
 struct another_stupid_type {
-	rofl_t blergh_gnampf;  //!< variables and functions in lowercase
-	                       //   with _ as word separator.
-	epictype42_t løl;      //!< no question, we use UTF-8 encoding.
-	ub0rtype røfëlkøptør;  //!< proper documentation is Doxygen parsable.
+	rofl_t blergh_gnampf;     //!< variables and functions in lowercase
+	                          //   with _ as word separator.
+	epictype42_t løl;         //!< no question, we use UTF-8 encoding.
+	ub0rtype     røfëlkøptør; //!< proper documentation is Doxygen parsable.
 };
 
 
-
 /**
  * Yet another Valve game. Go on, nothing to see here.
  */
 class HalfLife3 : public ValveGame {
-// vvv The access modifier is at the same indent level as the class definition:
+	// vvv The access modifier is at the same indent level as the class definition:
 public:
-	void release_game(const ValveTeam &team,
+	void release_game(const ValveTeam& team,
```

Comment:
we place `*` and `&` to the variable name, not the type, since semantically it's part of the name, and not the type.
`int* lol, rofl;` declares lol as `int*` and rofl as `int`, so when written like this: `int *lol, rofl;` this is directly visible.